fence-agents.spec.in

```diff
@@ -758,6 +758,8 @@ Fence agents for devices with IPMI interface.
 %{_mandir}/man8/fence_ilo4.8*
 %{_sbindir}/fence_ilo5
 %{_mandir}/man8/fence_ilo5.8*
+%{_sbindir}/fence_ipmitool
```

Comment:
You need to update it in the spec-file too, and remove tests/data/metadata/fence_ipmitool.xml.

Then run make xml-upload and add the updated metadata too.